### PR TITLE
[DST-92]: fix styles for `Input` of type `file` & `color`

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

**Comment by OsamaAbdellateef** [comment]
> how can I test?

@sarahgm 
added some stories for them so you can test them and i will delete them later

**Comment by OsamaAbdellateef** [comment]
> how can I test?

I

> how can I test?

@sarahgm 
I have added more types, you can change them from storybook options for input `type `

File: packages/components/src/Input/Input.stories.tsx [hunk 1]
```diff
@@ -26,6 +26,8 @@ const meta = {
         'time',
         'url',
         'week',
+        'file',
```
Comment (sebald) [inline packages/components/src/Input/Input.stories.tsx:29]: Can you also make a separate story to show them! :)

**Review by sebald — CHANGES_REQUESTED** [review]

File: packages/components/src/Input/Input.stories.tsx (hunk 1 above)
Comment (OsamaAbdellateef) [reply packages/components/src/Input/Input.stories.tsx:29]: @sebald 
already did that in this commit but i deleted them  :D 
https://github.com/marigold-ui/marigold/pull/3244/commits/e1dc6ab8c0fa08e27e01496695c16f1fdf14527a
i will get them back :100:

**Review by sarahgm — APPROVED** [review]

**Pull request merged**